core/src/trezor/lvglui/i18n/locales/en.py

```
@@ -870,9 +869,86 @@
+    "Fingerprint Upgrade",
+    "Set up now",
+    "Put your finger on the power button and lift it afterwards, Move your finger as guided by the light dot, repeating this step.",
+    "No Passphrase Linked to This PIN",
+    "Attach Passphrase",
+    "PIN has attached one Passphrase",
+    "You can attach a passphrase to this PIN.",
+    "You can remove this PIN or attach a new Passphrase.",
+    "PIN Already Used",
+    "This PIN has already been used. Please try a different one.",
+    "Hit the Limit",
+    "You’ve reached the maximum of 30 PINs. Please remove some before adding new ones.",
```

Ruff (0.11.9)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)